contract_api/registry.py

```diff
@@ -240,6 +240,27 @@ def get_group_info(self, org_id, service_id):
             print(repr(e))
             raise e
 
+    def get_service_media(self,org_id,service_id):
+        try:
+            query = """select `row_id`,url,`order`,file_type,asset_type,alt_text from service_media 
+            where service_id = %s and org_id = %s
+            order by `order` """
```

Comment:
Do you need to order? Order by will slow down the query

Reply:
this is addressed